compaction/compaction_manager.hh

```diff
@@ -476,6 +477,14 @@ public:
 
     virtual ~compaction_task_executor() = default;
 
+    // Compacts set of SSTables according to the descriptor.
+    // called when a compaction replaces the exhausted sstables with the new set
```

Comment:
the first sentence seems wrong, this interface is only about updating metadata on sstable replacement

Reply:
can be fixed in a followup.